oci/oci.go

```diff
@@ -515,7 +515,9 @@ func (r *Runtime) StopContainer(c *Container, timeout int64) error {
 	}
 	done := make(chan struct{})
 	go func() {
-		for {
+		ticker := time.NewTicker(time.Duration(timeout) * time.Second)
```

Comment:
Doesn't seem correct to me. Does this mean we'll at least wait for `timeout`?

What if the container exits immediately? A short interval (100 ms or something) may be better?